src/lib/provable/crypto/foreign-ecdsa.ts

```diff
@@ -99,19 +99,61 @@ class EcdsaSignature {
    * let isValid = sig.verify(msg, pk);
    * isValid.assertTrue('signature verifies');
    * ```
+   * @deprecated There is a security vulnerability in this method. Use {@link verifyV2} instead.
```

Comment:
could you completely remove any other doccomments here except the deprecation notice? we don't want to encourage people to use it

Reply:
[Fixed](https://github.com/o1-labs/o1js/pull/1669/commits/4a659c807188d05d378d2b28d200ae5a03d6900e)